kernel/kernel-biometrics-api/src/main/java/io/mosip/kernel/biometrics/commons/CbeffValidator.java

```diff
@@ -60,7 +60,8 @@ public static boolean validateXML(BIR birRoot) throws CbeffException {
 		List<BIR> birList = birRoot.getBirs();
 		for (BIR bir : birList) {
 			if (bir != null) {
-				if (bir.getBdb().length < 1) {
+				if ((bir.getBdb()==null || bir.getBdb().length < 1)
+						&& (bir.getOthers()==null || bir.getOthers().get("EXCEPTION").equals(false))) {
```

Comment:
When exception will be true? when a applicant has exception in one or few attributes of a modality or he has exception in a all modalities ?

Reply:
Here one BIR has been considered as only one Bio-attribute.

Ex : Left Middle Finger

Reply:
Also handle the case of .get("EXCEPTION") comming as null